source/jormungandr/jormungandr/transient_socket.py

```diff
@@ -157,7 +157,7 @@ def call(self, content, timeout, debug_cb=lambda: "", quiet=False):
             raise e
         except:
             self.close_socket(timed_socket.socket)
-            self._logger.exception("")
+            self._logger.exception("Unexpected transient socket exception")
```

Comment:
maybe also log `self.name`, `self._zmq_socket` and `debug_cb()` ?

Reply:
As discussed, all the information are logged and these words added will help to find those logged messages.

Reply:
@pbench's comment is pertinent.

```suggestion
            self._logger.exception("Unexpected transient socket exception with coverage: %s,  zmq_socket: %s, debug_info: %s", self.name, self._zmq_socket, debug_cb())
```

Reply:
OK, it may help indeed, didn't think of that.
This is also worth doing for the other change I think.

Reply:
As I think more: it may help even more around DeadSocketExceptions, no?